docker-vms/freeswitch/Dockerfile

```diff
@@ -0,0 +1,25 @@
+# Taken from https://freeswitch.org/confluence/display/FREESWITCH/Debian#Debian-DebianPackage
+
+FROM debian:wheezy
+MAINTAINER Praekelt Foundation <dev@praekeltfoundation.org>
+
+# Add the repository
+RUN echo 'deb http://files.freeswitch.org/repo/deb/debian/ wheezy main' \
+    >> /etc/apt/sources.list.d/freeswitch.list
+RUN gpg --keyserver pool.sks-keyservers.net --recv-key D76EDC7725E010CF
+RUN gpg -a --export D76EDC7725E010CF | apt-key add -
+
+# Install the packages
+ADD retry_until_pass.sh ./
+RUN ./retry_until_pass.sh apt-get update
```

Comment:
Added a comment in the Dockerfile. (not sure if it should be there or in the script)